frontend/public/components/masthead-toolbar.jsx

```diff
@@ -153,6 +170,8 @@ class MastheadToolbar_ extends React.Component {
   _renderMenu(mobile) {
     const { flags } = this.props;
     const { isUserDropdownOpen, isKebabDropdownOpen, username } = this.state;
+    const helpActions = this._helpActions();
```

Comment:
Not sure whats the benefit of initialising an extra `const` instead of calling the methods directly.